testsuite/tests/lib-dynlink-domains/main.ml

```diff
@@ -0,0 +1,357 @@
+all_modules = "Plugin_1_1.ml"
+************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_1_2.cmxs"
+module = ""
+all_modules = "Plugin_1_2.ml"
+*************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_1_2_0.cmxs"
+module = ""
+all_modules = "Plugin_1_2_0.ml"
+**************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_1_2_0_0.cmxs"
+module = ""
+all_modules = "Plugin_1_2_0_0.ml"
+***************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_1_3.cmxs"
+module = ""
+all_modules = "Plugin_1_3.ml"
+****************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_2.cmxs"
+module = ""
+all_modules = "Plugin_2.ml"
+******************************* ocamlopt.byte
+flags = "-shared"
+program= "Plugin_2_0.cmxs"
+module = ""
+all_modules = "Plugin_2_0.ml"
+******************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_2_0_0.cmxs"
+module = ""
+all_modules = "Plugin_2_0_0.ml"
+********************************* ocamlopt.byte
+flags = "-shared"
+program= "Plugin_2_0_0_0.cmxs"
+module = ""
+all_modules = "Plugin_2_0_0_0.ml"
+********************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3.cmxs"
+module = ""
+all_modules = "Plugin_3.ml"
+*********************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3_0.cmxs"
+module = ""
+all_modules = "Plugin_3_0.ml"
+************************************ ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3_1.cmxs"
+module = ""
+all_modules = "Plugin_3_1.ml"
+************************************* ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3_2.cmxs"
+module = ""
+all_modules = "Plugin_3_2.ml"
+************************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3_2_0.cmxs"
+module = ""
+all_modules = "Plugin_3_2_0.ml"
+*************************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3_2_0_0.cmxs"
+module = ""
+all_modules = "Plugin_3_2_0_0.ml"
+**************************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3_2_1.cmxs"
+module = ""
+all_modules = "Plugin_3_2_1.ml"
+***************************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3_2_2.cmxs"
+module = ""
+all_modules = "Plugin_3_2_2.ml"
+****************************************** ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3_2_2_0.cmxs"
+module = ""
+all_modules = "Plugin_3_2_2_0.ml"
+******************************************* ocamlopt.byte
+flags = "-shared"
+program= "Plugin_3_2_2_1.cmxs"
+module = ""
+all_modules = "Plugin_3_2_2_1.ml"
+******************************************** ocamlopt.byte
+flags = ""
+module = "main.ml"
+********************************************* ocamlopt.byte
+program = "./main.exe"
+libraries="dynlink"
+all_modules = "store.cmx main.cmx"
+module = ""
+********************************************** run
+*********************************************** check-program-output
```

Comment:
Can you make this more readable? :smiling_imp:

Reply:
I can certainly simplify the code generator to output readable numbers since you asked so vehemently :stuck_out_tongue: .